ipdb/__main__.py

```diff
@@ -25,10 +25,12 @@ def import_module(possible_modules, needed_module):
                 raise
 try:
     # IPython 5.0 and newer
-    from IPython.terminal.debugger import TerminalPdb as Pdb
     from IPython.core.debugger import BdbQuit_excepthook
+    from IPython.terminal.interactiveshell import TerminalInteractiveShell
+    debugger_cls = TerminalInteractiveShell().debugger_cls
```

Comment:
alternatively I was going to define it as a function rather than at import time, which now that I think about it, sounds much better.